rust-runtime/aws-smithy-types/Cargo.toml

```diff
@@ -11,17 +11,30 @@ license = "Apache-2.0"
 repository = "https://github.com/awslabs/smithy-rs"
 
 [features]
+rt-tokio = ["dep:tokio-util", "dep:tokio", "tokio?/rt", "tokio?/fs", "tokio?/io-util", "tokio-util?/io"]
 test-util = []
 serde-serialize = []
 serde-deserialize = []
 
 [dependencies]
 base64-simd = "0.8"
+bytes = "1"
+bytes-utils = "0.1"
+http = "0.2.3"
+http-body = "0.4.4"
+hyper = "0.14.26"
 itoa = "1.0.0"
 num-integer = "0.1.44"
+pin-project-lite = "0.2.9"
+pin-utils = "0.1.0"
```

Comment:
We might be able to get rid of pin-utils at this point due to more recent stabilizations.

Reply:
Thanks for the tip! Since this PR only handles the move, we can clean them up in a subsequent PR.

Reply:
I've searched for it, but did not find a likely feature. Let me know if you recall it.